changelog/8259.improvement.md

```diff
@@ -0,0 +1 @@
+Changes and additional steps in the section of the `README.md` file that describes the release process. 
```

Comment:
In my opinion we don't need a changelog for this as it doesn't affect the product usage. If you want to add one then I'd make it a `misc` changelog.